templates/search/catalogue_search.html

```diff
@@ -31,48 +43,63 @@
             {% include './blocks/visualisation_menu.html' %}
         {% endif %}
 
-        {% if buckets.current.result_count > 0 %}
+        {# Map view is displayed without results and filters #}
+        {% if form.views.value == views.MAP.value %}
             <div class="tna-container">
                 <div class="tna-column tna-column--full">
-                    {% include './blocks/search_sort_and_view_options.html' %}
+                    {% include './blocks/search_results_map.html' %}
                 </div>
             </div>
-        {% endif %}
-
-        {% if buckets.current.result_count > 0 or form.errors %}
-            <div class="tna-container tna-!--margin-top-s" data-id="catalogue-search-grid">
-                <div class="tna-column tna-column--width-1-3 tna-column--full-medium tna-column--full-small tna-column--full-tiny tna-!--margin-top-s" data-id="catalogue-search-sidebar">
-                    {% include './blocks/search_filters.html' %}
+        {% else %}
+            {% if buckets.current.result_count > 0 %}
+                <div class="tna-container">
+                    <div class="tna-column tna-column--full">
```

Comment:
same as above - Why are the div tna-container, tna-column tna-column--full - outside of block html? Is there any issue to include then inside the block html?